src/main/java/com/fasterxml/jackson/module/jsonSchema/customProperties/SchemaPropertyProcessorManagerFactoryWrapper.java

```diff
@@ -0,0 +1,89 @@
+package com.fasterxml.jackson.module.jsonSchema.customProperties;
+
+import com.fasterxml.jackson.databind.BeanProperty;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.jsonFormatVisitors.JsonObjectFormatVisitor;
+import com.fasterxml.jackson.module.jsonSchema.JsonSchema;
+import com.fasterxml.jackson.module.jsonSchema.factories.ObjectVisitor;
+import com.fasterxml.jackson.module.jsonSchema.factories.ObjectVisitorDecorator;
+import com.fasterxml.jackson.module.jsonSchema.factories.SchemaFactoryWrapper;
+import com.fasterxml.jackson.module.jsonSchema.factories.VisitorContext;
+import com.fasterxml.jackson.module.jsonSchema.factories.WrapperFactory;
+import com.fasterxml.jackson.module.jsonSchema.factories.WrapperFactory.JsonSchemaVersion;
+import com.fasterxml.jackson.module.jsonSchema.property.manager.SchemaPropertyProcessorManagerApi;
+import com.fasterxml.jackson.module.jsonSchema.types.ObjectSchema;
+
+/**
```

Comment:
Could you add Javadoc comments on puprose of this class?

Reply:
I will certainly add javadoc. Please point out other places that are lacking as well and I will get them too. 
I am not a fan of the name at all. Any suggestions on a better one?